tests/Paket.Tests/Paket.Tests.fsproj

```diff
@@ -261,9 +265,16 @@
     <Compile Include="InstallModel\Penalty\FrameworkConditionsSpecs.fs" />
     <Compile Include="DependencyModel\ProjectDependencySpecs.fs" />
     <None Include="paket.references" />
+    <Content Include="App.config" />
   </ItemGroup>
   <ItemGroup>
     <Reference Include="mscorlib" />
+    <Reference Include="nunit.core">
```

Comment:
On mobile now so don't have a good overview, but think that's wrong, should be reference/content or whatever the canonical way of doing it is. Would you mind opening an issue?